.travis.yml

```diff
@@ -4,17 +4,34 @@ sudo: false
 cache:
   - pip
 
+before_cache:
+  - rm ~/.cache/pip/log/debug.log
+
 language: python
 python:
   - "2.7"
   - "3.5"
 
 install:
-  - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
-  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
-  - ansible-playbook --syntax-check playbooks/byo/config.yml
   - tox
-  - cd utils && tox
+
+notifications:
+  email:
+    recipients:
+      - jdetiber@redhat.com
+      - sdodson@redhat.com
+    on_success: change
+    on_failure: always
+  irc:
+    channels:
+      - chat.freenode.net#openshift-dev
+    on_success: change
+    on_failure: always
+    template:
+      - "%{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message}"
+      - "Change view : %{compare_url}"
+      - "Build details : %{build_url}"
+      - "sdodson jdetiber: ^"
```

Comment:
notifications are for giving us an overview when the build is broken post-merge, since we've had some issues with generate_validation being broken after a merge.